external/mcpclient/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@microsoft/spark.mcpclient",
```

Comment:
wondering if there is a way we can combine the client/server into one package?

Reply:
I think it's possible, but it does conflate the purpose of prompts a bit, so it might get a bit confusing maybe?

Reply:
Talked offline. Let's keep them separate for now. If a server wants to do both, it can compose it using these packages.

Question: Should I rename `mcp` to `mcpServer`?

Reply:
I don't have strong feelings either way, Socketio does it the way we have it here (naming wise) but I could also see it being the other way.